src/ts/components/transaction/Send.tsx

```diff
@@ -87,18 +87,18 @@ class Send extends React.Component<ISendProps, ISendState> {
 
   changeAddress = event => {
     const val = event.target.value
```

Comment:
perhaps remove this const declaration and use `event.target.value` directly just like `changeAmount` and `changeFee`

Reply:
okay removing a leftover after testing